source/styles.css

```diff
@@ -21,3 +22,6 @@
 .VirtualizedSelectSelectedOption {
   font-weight: bold;
 }
+.Select div {
+    box-sizing: content-box !important;
```

Comment:
I don't think I'm comfortable with this. In general, `!important` rules are kind of gross– and changing the `box-sizing` might break existing custom styles.

Reply:
Agree. Is there any way of implementing the `box-sizing` in a safer way?

Reply:
Leave it up to applications to specify their own styles. (I just don't think this should be in the library as a default)